web/src/hooks/useLogActionsExtension.tsx

```diff
@@ -77,7 +85,14 @@ const useLogActionsExtension: ExtensionHook<Array<Action>, LogActionsExtensionOp
               const params = new URLSearchParams();
               params.set('q', parsedQuery.toString());
               params.set('tenant', tenant);
-              const href = `/monitoring/logs?${params.toString()}`;
+              let href = `/monitoring/logs?${params.toString()}`;
+
+              if (perspective === 'dev') {
+                const namespaceToUse = alertNamespace || activeNamespace;
+                href = `/dev-monitoring/ns/${namespaceToUse}/logs?${params.toString()}`;
+              } else {
+                href = `/monitoring/logs?${params.toString()}`;
```

Comment:
Either the initial setting of href can be removed or this else can be removed

Reply:
yes indeed, I removed the initial value